msteams-platform/resources/schema/manifest-schema-dev-preview.md

```diff
@@ -997,12 +997,13 @@ The `extensions` property specifies Outlook Add-ins within an app manifest and s
 |`alternates`| Array | 10 | | Specifies the relationship to alternate existing Microsoft 365 solutions. It's used to hide or prioritize add-ins from the same publisher with overlapping functionality. |
 |`audienceClaimUrl`| String | 2048 characters | | Specifies the URL for your extension and is used to validate Exchange user identity tokens. For more information, see [inside the Exchange identity token](/office/dev/add-ins/outlook/inside-the-identity-token)|
 |`appDeeplinks`| Array | | | Do not use. For Microsoft internal use only. |
+|`getStartedMessages`| Array | 3 | | Provides information about an Office Add-in that appears in various places in Office, such as the callout that appears in Office when an Office Add-in is installed. |
```

Comment:
Hi @Rick-Kirkham,
For this line Office Add-in that appears in various places in Office, it will be clearer if we explain what the various places are.

Reply:
> Hi @Rick-Kirkham, For this line Office Add-in that appears in various places in Office, it will be clearer if we explain what the various places are.

Actually, the callout is the only place currently where it appears, so I'll just remove the bit about "various places".